geometry_msgs/Pose pose

# old index before sorting
uint32 oldIdx

# weighted rating
float32 rating
# utility and costs
float32 utility
float32 inverse_costs
# specific costs
float32 base_translation_inverse_costs
float32 base_rotation_inverse_costs
float32 ptu_movement_inverse_costs
float32 recognition_inverse_costs
